src/wallet/wallet_rpc_server.cpp

```diff
@@ -4353,21 +4363,31 @@ namespace tools
       return false;
     }
 
-    return fill_response(ptx_vector,
-                         req.get_tx_key,
-                         res.tx_key,
-                         res.amount,
-                         res.fee,
-                         res.multisig_txset,
-                         res.unsigned_txset,
-                         req.do_not_relay,
-                         false /*blink*/,
-                         res.tx_hash,
-                         req.get_tx_hex,
-                         res.tx_blob,
-                         req.get_tx_metadata,
-                         res.tx_metadata,
-                         er);
+    try
+    {
+      return fill_response(ptx_vector,
+                           req.get_tx_key,
+                           res.tx_key,
+                           res.amount,
+                           res.fee,
+                           res.multisig_txset,
+                           res.unsigned_txset,
+                           req.do_not_relay,
+                           false /*blink*/,
+                           res.tx_hash,
+                           req.get_tx_hex,
+                           res.tx_blob,
+                           req.get_tx_metadata,
+                           res.tx_metadata,
+                           er);
+    }
+    catch (const std::exception &e)
+    {
+      handle_rpc_exception(std::current_exception(), er, WALLET_RPC_ERROR_CODE_GENERIC_TRANSFER_ERROR);
+      return false;
+    }
+
+    return true;
```

Comment:
This statement is unreachable: the code either returns from inside the try or inside the catch (or else, in theory, throws if given some non-std::exception-derived thrown value).

Reply:
yeah removed.